### ISSUE TITLE
Receive prop if scrollbars are on/off

### ISSUE THREAD

**Issue by seleckis** [malte-wessel/react-custom-scrollbars#107]
I use Scrollbars in responsive design. But I need to show an icon when scrolling is available. But it seems I cannot get any changed prop when screen size has been changed. Is it possible to pass prop like `isScrollable` from Scrollbars component to nested component?

**Comment by malte-wessel** [comment]
You could reference the `Scrollbars` component and check if the `scrollHeight` value is greater than `0`
```javascript
const MyComponent = createClass({
  getInitialState() {
    return {
      isScrollingAvailable: true
    };
  },
  componentDidMount() {
    this.updateIsScrollingAvailable();
  },
  componentDidUpdate() {
    this.updateIsScrollingAvailable();
  },
  updateScrollingAvailable() {
    const { scrollbars } = this.refs;
    const { scrollHeight } = scrollbars.getValues();
    const { isScrollingAvailable } = this.state;
    const nextIsScrollingAvailable = scrollHeight > 0;
    if (nextIsScrollingAvailable !== isScrollingAvailable) {
      this.setState({ isScrollingAvailable: nextIsScrollingAvailable });
    }
  }
  render() {
    return (
      <Scrollbars ref="scrollbars">
         <div>Some long content...</div>
         <div>
           {this.state.isScrollingAvailable ? 'Scrolling available' : 'Scrolling not available'}
         </div>
      </Scrollbars>
    );
});
```

**Comment by seleckis** [comment]
Well, I think it's worth to include that into this package instead of use this code every time. This is not special useage case, but trivial functionality, isn't it?

**Comment by malte-wessel** [comment]
I don't think that this is a common use case. I never had to use that functionality and I think it's fairly easy to implement this in user land

**Comment by seleckis** [comment]
Don't you think that scrollHeight will return 0 when scroll position is at the top? And what about responsive design? Code will be much more complex, if we need to listen to window resize.

**Comment by malte-wessel** [comment]
The `scrollHeight` value is the height of the element plus its overflow. So if you have an element that is 100px height (`overflow-y: scroll`) and has a child that is 300px height, `scrollHeight` would be 300px, no matter what the scroll position is. (Btw I updated the example, I checked mistakenly for `scrollHeight > 0` which should be `(scrollHeight - clientHeight) > 0`.

Regarding the window resize handler, this is fairly easy to implement. Just implement your custom `Scrollbars` component if you want to reuse the functionality.
